README.md

```diff
@@ -87,11 +89,27 @@ import { AppComponent } from './app.component';
     BrowserModule
   ],
   providers: [],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA],
-  bootstrap: [AppComponent]
+  schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
+  bootstrap: [ AppComponent ]
 })
 export class AppModule { }
 
+// or the vanilla JS version
+
+(function(app) {
+  app.AppModule =
+    ng.core.NgModule({
+      imports: [ ng.platformBrowser.BrowserModule ],
+      declarations: [ app.AppComponent ],
+      schemas: [ ng.core.CUSTOM_ELEMENTS_SCHEMA ],
+      bootstrap: [ app.AppComponent ]
+    })
+    .Class({
+      constructor: function() {}
+    });
+})(window.app || (window.app = {}));
+
+
 /*
 You can now use the custom element in either a standalone or inline template
```

Comment:
Yeah, that boilerplate is completely Angular related. Every Angular application will already have at least one NgModule. 

So, the steps are:
1) Use the Custom Element adapter and register the element (which is already available)
2) Literally, add 2 lines to the already existing NgModule in your application.
3) You're ready to go since Angular is able to deal with custom elements natively.

I really can't see a more straightforward process than that. The other option would be to implement a directive, component or service but that would only add more complexity since now you'd have to provide the component, make the service injectable, etc (all of this is Angular related) which would also mean more boilerplate to set up properly. 

With the current approach, we leverage Angular native capabilities and embrace the standard, hence future-proofing our current solution.

I guess the main issue is that Angular is not like any of the other frameworks in the Readme, we're not even "registering" the component because there's no need to. We can just leverage the Custom Element adapter and we're ready to go.